Dockerfile

```diff
@@ -21,16 +19,28 @@ COPY main.go main.go
 COPY api/ api/
 COPY pkg/ pkg/
 
-# Build
-RUN . ${ENVFILE}; CGO_ENABLED=0 GOOS=linux GOARCH=${ARCH} GO111MODULE=on make build
+# Build (amd64)
+WORKDIR /workspace-amd64
+RUN cp -r /workspace/* .
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on make build
 
+# Build (s390x)
+WORKDIR /workspace-s390x
+RUN cp -r /workspace/* .
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=s390x GO111MODULE=on make build
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal
-
-RUN microdnf update -y && microdnf clean all
 
+# amd64 target image
+FROM --platform=amd64 registry.access.redhat.com/ubi8/ubi-micro as amd64-build
```

Comment:
Can the `--platform` be a parameter passed to podman when building? We could then avoid code duplication.

Reply:
Yes, according to the docs this is possible.

Reply:
Done.